tests/test_harvest/test_mit_harvester.py

```diff
@@ -133,6 +133,25 @@ def test_mit_harvester_incremental_harvest_two_zip_files_returned(
         assert len(list(records)) == 2  # noqa: PLR2004
 
 
+def test_mit_harvester_incremental_continues_after_missing_zip_file(
+    caplog,
+    mock_sqs_queue,
+    mocked_sqs_topic_name,
+    mocked_restricted_bucket_one_legacy_fgdc_zip,
+):
+    harvester = MITHarvester(
+        harvest_type="incremental",
+        input_files=mocked_restricted_bucket_one_legacy_fgdc_zip,
+        sqs_topic_name=mocked_sqs_topic_name,
+    )
+    records = harvester.incremental_harvest_get_source_records()
+    assert len(list(records)) == 1
+    assert (
+        "OSError: unable to access bucket: 'mocked_cdn_restricted' "
+        "key: 'cdn/geo/restricted/DEF456.zip'" in caplog.text
+    )
```

Comment:
For this test, I'll admit I had to do some digging to understand why it worked.  What's not immediately obvious to me is that mocked queue contains _two_ messages: one referencing a file that is in the mocked S3 bucket, and one that is not.  

Instead of a docstring, what about updating the fixture `mock_sqs_queue` to something like `mock_sqs_queue_two_messages`?  I feel like the "two" in that fixture and "one" from `mocked_restricted_bucket_one_legacy_fgdc_zip` would be enough to understand that final `assert = 1` for the length of the results --> one failed, but one still succeeded.

Not a blocking request, but just sharing my experience unpacking this test (which is good, BTW).

Reply:
Update: see comment above about yield Record + exception from the `incremental_harvest_get_source_records()` method.  If that approach is used, this test would need a couple of updates and maybe then the fixture name is more inconsequential.